docs/Pubsub.java

```diff
@@ -0,0 +1,49 @@
+package com.spaceuptech;
+
+import com.spaceuptech.space_api.API;
+import com.spaceuptech.space_api.utils.*;
+
+import java.util.HashMap;
+
+public class Pubsub {
+    public static void main(String[] args) throws Exception {
+        API api = new API("books-app", "localhost", 4124);
+        Pubsub pubsub = api.pubsub();
+        PubsubSubscription subscription = pubsub.subscribe("/subject/", new PubsubListener() {
+            @Override
+            public void onReceive(String subject, Object data) {
```

Comment:
`data` should be of type Response. It should have a get value function which takes a type and returns the value in that type

Reply:
Done